src/neuroconv/datainterfaces/ophys/requirements.txt

```
@@ -1 +1 @@
-roiextractors>=0.5.2
+roiextractors @ git+https://github.com/catalystneuro/roiextractors.git@51ccccbb53fbab7e389188b60693b0e5633746bf#egg=roiextractors
```

```suggestion
roiextractors>=0.5.3
```

Went ahead and cut a new release for this